modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -252,4 +262,235 @@ void EditorSceneFormatImporterBlend::get_import_options(const String &p_path, Li
+void EditorFileSystemImportFormatSupportQueryBlend::_path_confirmed() {
+	confirmed = true;
+}
+
+void EditorFileSystemImportFormatSupportQueryBlend::_select_install(String p_path) {
+	blender_path->set_text(p_path);
+	_validate_path(p_path);
+}
+void EditorFileSystemImportFormatSupportQueryBlend::_browse_install() {
+	if (blender_path->get_text() != String()) {
+		browse_dialog->set_current_dir(blender_path->get_text());
+	}
+
+	browse_dialog->popup_centered_ratio();
+}
+
+bool EditorFileSystemImportFormatSupportQueryBlend::query() {
+	if (!configure_blender_dialog) {
+		configure_blender_dialog = memnew(ConfirmationDialog);
+		configure_blender_dialog->set_title(TTR("Configure Blender Importer"));
+		configure_blender_dialog->set_flag(Window::FLAG_BORDERLESS, true); // Avoid closing accidentally .
+		configure_blender_dialog->set_close_on_escape(false);
+
+		VBoxContainer *vb = memnew(VBoxContainer);
+		vb->add_child(memnew(Label(TTR("Blender 3.0+ is required to import '.blend' files.\nPlease provide a valid path to a Blender installation:"))));
+
+		HBoxContainer *hb = memnew(HBoxContainer);
+
+		blender_path = memnew(LineEdit);
+		blender_path->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+		hb->add_child(blender_path);
+		blender_path_browse = memnew(Button);
+		hb->add_child(blender_path_browse);
+		blender_path_browse->set_text(TTR("Browse"));
+		blender_path_browse->connect("pressed", callable_mp(this, &EditorFileSystemImportFormatSupportQueryBlend::_browse_install));
+		hb->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+		hb->set_custom_minimum_size(Size2(400 * EDSCALE, 0));
+
+		vb->add_child(hb);
+
+		path_status = memnew(Label);
+		vb->add_child(path_status);
+
+		configure_blender_dialog->add_child(vb);
+
+		blender_path->connect("text_changed", callable_mp(this, &EditorFileSystemImportFormatSupportQueryBlend::_validate_path));
+
+		EditorNode::get_singleton()->get_gui_base()->add_child(configure_blender_dialog);
+
+		configure_blender_dialog->get_ok_button()->set_text(TTR("Confirm Path"));
+		configure_blender_dialog->get_cancel_button()->set_text(TTR("Disable '.blend' Import"));
+		configure_blender_dialog->get_cancel_button()->set_tooltip(TTR("Disables blender import for this project. Can be re-enabled in Project Settings"));
+		configure_blender_dialog->connect("confirmed", callable_mp(this, &EditorFileSystemImportFormatSupportQueryBlend::_path_confirmed));
+
+		browse_dialog = memnew(EditorFileDialog);
+		browse_dialog->set_access(EditorFileDialog::ACCESS_FILESYSTEM);
+		browse_dialog->set_file_mode(EditorFileDialog::FILE_MODE_OPEN_DIR);
+		browse_dialog->connect("dir_selected", callable_mp(this, &EditorFileSystemImportFormatSupportQueryBlend::_select_install));
+
+		EditorNode::get_singleton()->get_gui_base()->add_child(browse_dialog);
+	}
+
+	String path = EDITOR_GET("filesystem/import/blender/blender3_path");
+
+	if (path == "") {
+		// Autodetect
+		auto_detected_path = "";
+
+#if defined(OSX_ENABLED)
+
+		// MacOS Detection
```

Comment:
- Execute `mdfind kMDItemCFBundleIdentifier="org.blenderfoundation.blender"` command and append `/Contents/MacOS/Blender` (as the main detection method).
- If failed, following locations can be checked: `/opt/homebrew/bin/blender`, `/opt/local/bin/blender`, `/usr/local/bin/blender`, `/usr/local/opt/blender` and `/Applications/Blender.app/Contents/MacOS/Blender` .

Reply:
I can't currently test, so probably better if you add this as an additional PR after this one is merged.

Reply:
https://github.com/godotengine/godot/pull/59773